source/LoadInsight.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+
+def main() :
+	"""Dummy main to test basic pipeline functionality"""
+	input1 = DataObject()
+	output1 = DataObject()
+	transform = Transformation(inputs=[input1],outputs=[output1],function=nop)
+	p1 = Pipeline()
+	p1.add(transform)
+	p1.run()
+
+def nop(x):
+	return(x)
+
+class DataObject:
+	"""DataObject hold all data artifacts as pandas DataFrames 
+	   and associated storage specifications.
+	"""
+	local_storage = {"path":""}
+	remote_storage = {} # TODO: from infrastucture
+
+	def __init__(self,
+			name="NULL",
+			data=pd.DataFrame(),
+			url=None,
+			convert=None) :
+	
+		# this is the dataframe to use
+		self.name = name
+		self.data = data
+	
+		if url != None :
+			import curl
```

Comment:
we shouldn't use `curl`, `requests` is much more robust. Also, we'll move this class anyways, but I'd refrain from these style of imports and put them at the top level.